target/linux/x86/base-files/usr/sbin/grow_rootfspart

```diff
@@ -0,0 +1,327 @@
+#!/bin/sh
+# shellcheck disable=SC2039
+#
+# Grow partition 2 (rootfs_data) until the end of the disk
+#
+# 
+
+readbin2hex() {
+	local input=$1
+	local seek=$2
+	local length=$3
+	hexdump -ve '1/1 "%.2x"' -s "$seek" -n "$length" "$input"
+}
+
+hex2bin(){
+	printf "%s\n" "$1" | sed -r 's/(.{2})/\\\\x\1 /g' | xargs printf '%b' 
+}
+
+writehex2bin(){
+	local output=$1
+	local seek=$2
+	local hexbuf=$3
+	local length=$((${#hexbuf}/2))
+	hex2bin "$hexbuf" | dd bs=1 seek="$seek" count="$length" of="$output" conv=notrunc 2>/dev/null
+}
+writehex2buf(){
+	local buf=$1
+	local seek=$2
+	local modified=$3
+	printf "%s" "${buf:0:seek*2}"
+	printf "%s" "$modified"
+	printf "%s\n" "${buf:seek*2+${#modified}}"
+}
+
+crc32_hex(){
+	gzip -c | tail -c8 | hexdump -n4 -e '"%.8x" "\n"'
+}
+
+
+msg(){
+        printf "grow_rootfspart: %s\n" "$*" >&2
+
+}
+die() {
+	local err=$1; shift
+	msg "$@"
+	exit "$err"
+}
+
+# swap endianness
+bswap(){
```

Comment:
I'd prefer to have a space between () and {
This is currently mixed in the file ...

Reply:
spaces added